### DISCUSSION TITLE
Bad translation on Chinese Simplified

### DISCUSSION THREAD

**Discussion by caic99** [community/community#43564] [Mobile]
Merge as administrator is translated to 合并为管理员, which does not match the original meaning.

My suggested translation would be 作为管理员合并.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.93.0 (85890385)
Device: iPhone 13 Pro (iOS 16.1.2)
</details>

**Comment by candyho** [comment]
Hey thanks for the feedback. I will review this with the internal team and work on a fix soon.
